src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj

```diff
@@ -14,6 +14,7 @@
     <DebugSymbols>true</DebugSymbols>
     <PlatformTarget>AnyCPU</PlatformTarget>
     <ProjectGuid>{9E50E7BF-CD6E-4269-A6DD-59FD0BD6C0FD}</ProjectGuid>
+    <PackageTargetFramework Condition="'$(TargetGroup)' == ''">DNXCore50</PackageTargetFramework>
```

Comment:
Sounds good. @StephenBonikowsky can you please be sure to file issues tracking those updates?